acceptance/analyzer_test.go

```diff
@@ -70,7 +71,10 @@ func TestAnalyzer(t *testing.T) {
 	h.DockerBuild(t,
 		analyzeImage,
 		analyzeDockerContext,
-		h.WithFlags("-f", filepath.Join(analyzeDockerContext, variables.DockerfileName)),
+		h.WithFlags(
+			"--build-arg", fmt.Sprintf("cnb_platform_api=%s", api.Platform.Latest()),
```

Comment:
Now that we have acceptance tests for the restorer, I think it makes sense to test the latest platform api there was well. IIUC the restorer acceptance tests are exercising platform api 0.3 (since we don't set `CNB_PLATFORM_API` in the image). We'll probably want to update them so that they test 0.5 and "latest".